### PR TITLE
html-forms | Chris Kim

### PR DESCRIPTION
**Pull request by chris-ys-kim**
Screen Shot 2021-05-04 at 11 23 43 AM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <div class="form-example">
+    <label for="email">Email: </label>
+    <div><input type="text" name="name" id="name" required></div>
```
Comment (scbowler) [inline html-forms/index.html:12]: The `for` attribute of the label should match the `id` of the related input. Also the `name` attribute should match the data being collected, so here `"email"` would be better for the `name` attribute.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  </div>
+
+    <label for="email">Why is coding cool?: </label>
+    <div><select name="devices">
```
Comment (scbowler) [inline html-forms/index.html:16]: The label `for` attribute should match the `id` of the `select`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <option value="Select an option">Select an option</option>
```
Comment (scbowler) [inline html-forms/index.html:17]: For the "default" option we usually leave the `value` as an empty string

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>Favoriate Language (choose one)
+      <div><input type="radio" name="type" value="HTML" /> HTML
```
Comment (scbowler) [inline html-forms/index.html:24]: Make sure to use a `<label>` element for the text that labels the radio button. Make sure the `for` attribute of the label matches the `id` attribute of the related radio button.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <input type="radio" name="type" value="CSS" />
+      CSS
+      <input type="radio" name="type" value="Javascript" />
+      Javascript</div>
+    </div>
+
+  <div>Would you like to sign up for more information?</div>
```
Comment (scbowler) [inline html-forms/index.html:31]: Use label here, you can still wrap it in a div but the labeling text for an input should be inside a label element.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  <input type="checkbox" name="sign" value="Sign me up!" /> Sign me up!
+    <div>Comments</div>
```
Comment (scbowler) [inline html-forms/index.html:33]: Use label

**Review by scbowler — CHANGES_REQUESTED** [review]
Few changes needed, let me know if you have any questions.

**Comment by chris-ys-kim** [comment]
fixed.

**Comment by chris-ys-kim** [comment]
Screen Shot 2021-05-04 at 12 24 14 PM

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  <div>
+    <p>Would you like to sign up for more information?</p>
+<input type="checkbox" name="signup" value="yes" id="signup">
+<label for="signup">Sign me up!</label>
+</div>
+<div>
+  <label for="comments">Comments</label><br>
+  <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+</div>
+<input type="submit">
```
Comment (scbowler) [inline html-forms/index.html:43]: Make sure to wrap your form elements in a `<form>` element and keep an eye on your indenting.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback! Looks good 👍 

One FYI comment below, no further changes required.

**Pull request merged**